sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -896,6 +896,12 @@ func withPrivilegedMounts() oci.SpecOpts {
 			Source:      "/mnt",
 			Type:        "bind",
 		},
+		{
+			Options:     []string{"bind", "ro"},
```

Comment:
Don't we have to make this `exec`?

Reply:
Huh. It wasn't that way in the original change and seemed to work fine. I'll change it to be consistent with the rest